worker_types/gwci-linux/userdata

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+mkdir -p /home/ubuntu/generic-worker
+cd /home/ubuntu/generic-worker
+apt update
+apt upgrade -y
+apt install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
+apt update
+apt-cache policy docker-ce | grep -qF download.docker.com
+apt install -y docker-ce
+sleep 5
+systemctl status docker | grep "Started Docker Application Container Engine"
+usermod -aG docker ubuntu
+id -nG | grep docker
+curl -L https://github.com/taskcluster/generic-worker/releases/download/v15.1.2/generic-worker-simple-linux-amd64 > generic-worker
+curl -L https://github.com/taskcluster/taskcluster-proxy/releases/download/v5.1.0/taskcluster-proxy-linux-amd64 > taskcluster-proxy
+curl -L https://github.com/taskcluster/livelog/releases/download/v1.1.0/livelog-linux-amd64 > livelog
+chmod a+x generic-worker taskcluster-proxy livelog
+chown -R ubuntu:ubuntu /home/ubuntu/generic-worker
+./generic-worker --version
+./generic-worker new-ed25519-keypair --file ed25519.key
+echo '@reboot PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /home/ubuntu/generic-worker/generic-worker run --configure-for-aws --config /home/ubuntu/generic-worker/generic-worker.config >> /home/ubuntu/generic-worker/generic-worker.log 2>&1' | crontab -u ubuntu -
+# apt install -y gnome
```

Comment:
are we keeping these commented out bits for now?

Reply:
I guess I could put them somewhere else... I commented them out because I suspect at some point we'll want to run stuff inside an actual desktop environment, and I didn't want to lose the steps I'd already worked out... But maybe I'm just making a mess by leaving them in there.